corehq/apps/hqcase/api/updates.py

```diff
@@ -108,6 +108,7 @@ def _if_specified(value):
                     relationship=index.relationship
                 ) for name, index in self.indices.items()
             },
+            **get_kwargs('case_type', 'case_name', 'external_id', 'owner_id'),
```

Comment:
I'm not a huge fan of this change, as it breaks the grep test - searching for `self.external_id` no longer works, and it's harder to see at a glance what values are passed in and from where.